src/main/java/io/github/moderngamingworlds_mods/woodenutilities/common/init/ModBlocks.java

```diff
@@ -16,6 +17,13 @@ public class ModBlocks {
     public static final Block woodenTnt = new WoodenTntBlock();
     public static final Block woodcutter = new WoodcutterBlock();
 
-    public static final Block woodenFurnace = new WoodenFurnaceBlock();
-    public static final BlockEntityType<WoodenFurnaceBlockEntity> WOODEN_FURNACE = BlockEntityType.Builder.of(WoodenFurnaceBlockEntity::new, woodenFurnace).build(null);
+    public static final Block oakFurnace = new WoodenFurnaceBlock(WoodType.OAK);
+    public static final Block spruceFurnace = new WoodenFurnaceBlock(WoodType.SPRUCE);
+    public static final Block birchFurnace = new WoodenFurnaceBlock(WoodType.BIRCH);
+    public static final Block acaciaFurnace = new WoodenFurnaceBlock(WoodType.ACACIA);
+    public static final Block jungleFurnace = new WoodenFurnaceBlock(WoodType.JUNGLE);
+    public static final Block darkOakFurnace = new WoodenFurnaceBlock(WoodType.DARK_OAK);
+    public static final Block crimsonFurnace = new WoodenFurnaceBlock(WoodType.CRIMSON);
+    public static final Block warpedFurnace = new WoodenFurnaceBlock(WoodType.WARPED);
```

Comment:
Could maybe add something like `EnumObjects` for wood types if there will be more things in different wood types.

Reply:
You mean in the same way like LibX with the `DecoratedBlock`?

Reply:
No, like LibX with `EnumObjects`. Could either make a hardcoded list of `WoodType`s to apply or even make these dynamic based on the list of installed mods. Which would then make adding new wood types extremely easy.

Reply:
Easier Always better for end-user right?